pkg/kubelet/util/ipv6_others.go

```diff
@@ -0,0 +1,25 @@
+//go:build !linux && !windows
+// +build !linux,!windows
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// IsIPv6Supported determines whether the host supports ipv6.
+func IsIPv6Supported() (bool, error) {
+	return false, nil
```

Comment:
so we change the behavior of all "other" without any options to override it back to actually support IPv6?

Reply:
yes, it may be more reasonable since this will not be a regression.